src/views/preview/preview.jsx

```diff
@@ -513,8 +528,17 @@ const mapStateToProps = state => {
         userPresent;
     const authorPresent = projectInfoPresent && state.preview.projectInfo.author &&
         Object.keys(state.preview.projectInfo.author).length > 0;
+    const userOwnsProject = isLoggedIn && authorPresent &&
+        state.session.session.user.id === state.preview.projectInfo.author.id;
 
     return {
+        canAddToStudio: isLoggedIn && userOwnsProject,
+        canCreateNew: false,
```

Comment:
Isn't this always true?

Reply:
canCreateNew? It will be true, but currently we have File->new disabled, until that PR goes up.

Reply:
Um yes, we should keep it disabled until the GUI tells us what ID to switch to.